features/admin-file-attributes.feature

```diff
@@ -0,0 +1,142 @@
+    When I go to "/adminFile/update/"
+    Then I should see "Login"
+
+  @ok
+  Scenario: Go to a published dataset found in production-like database
+    Given I am not logged in to Gigadb web site
+    When I go to "dataset/100056"
+    And I should see "Termitomyces sp. J132 fungus genome assembly data."
+    And I follow "History"
+    Then I should see "History" tab with text "File Termitomyces_assembly_v1.0.fa.gz updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.cds.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.cds.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.gff updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+    And I should see "History" tab with text "File Termitomyces_gene_v1.0.pep.fa updated"
+
+  @ok @published
+  Scenario: Sign in as admin and visit admin file update page and see New Attribute, Edit, Delete, Save buttons
+    Given I sign in as an admin
+    When I am on "/adminFile/update/id/13973"
+    Then I should see a button "New Attribute"
+    And I should see a file attribute table
+      | Attribute Name | Value     | Unit |
+      | last_modified  | 2013-7-15 |      |
+    And I should see a button input "Edit"
+    And I should see a button input "Delete"
+    And I should see a button input "Save"
+
+  @ok @javascript @published
+  Scenario:  Go to a published dataset found in production-like database, should see the File Attribute Id of the last modified attribute
+    Given I sign in as an admin
+    When I go to "/adminFile/view/id/13973"
+    Then I should see field "File Attribute Id" with value "1 "
+
+  @ok @javascript @published
+  Scenario: Go to a published dataset found in production-like database, delete a last modified attribute
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/13973"
+    When I press "Delete"
+    Then I should not see "last_modified"
+    And I should not see "2013-7-15"
+    And I should not see a button "Delete"
+
+  @ok @javascript @published
+  Scenario: Go to a published dataset found in production-like database, delete a last modified attribute and save
+    Given I sign in as an admin
+    And I am on "/adminFile/update/id/13973"
+    When I press "Delete"
+    And I press "Save"
+    Then I am on "/adminFile/view/id/13973"
+    And I should see field "File Attribute Id" with empty value
+
+  @ok @javascript @published
+  Scenario: Go to a published dataset found in production-like database, delete a last modified attribute and check for history tab
+    Given I sign in as an admin
+    And I go to "/adminFile/update/id/13973"
+    And I should see a file attribute table
+      | Attribute Name | Value     | Unit |
+      | last_modified  | 2013-7-15 |      |
+    And I should see a button input "Delete"
```

Comment:
Do we need this step and the step above since it is already tested in `Scenario: Sign in as admin and visit admin file update page and see New Attribute, Edit, Delete, Save buttons`?

Reply:
This step is to check the history tab containing the delete message or not after a file attribute is deleted. So, I think it is needed.

Reply:
I don't mean the Scenario. I mean these 2 steps:
```
And I should see a file attribute table
       | Attribute Name | Value     | Unit |
       | last_modified  | 2013-7-15 |      |
     And I should see a button input "Delete"
```

They are the same in the above `Scenario: See Attribute, Edit, Delete, and Save buttons on admin file update page`.

Reply:
These two steps have been removed.